torch/profiler/profiler.py

```diff
@@ -604,13 +604,11 @@ def __init__(
 
     def __enter__(self):
         prof._enable_dynamo_cache_lookup_profiler(True)
-        prof._set_is_profiler_enabled(True)
```

Comment:
Why did you remove these?

Reply:
I moved it to autograd/profiler.py and initially forgot to remove it from this file.

afaik "profiler enabled" == "torch._C._profiler._enable_profiler() has been called", and all the callsites for `_enable_profiler()` are in autograd/profiler.py - so I moved these calls to autograd/profiler.py.

Reply:
ok! All good then!